app/routes.py

```diff
@@ -1 +1,217 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, request, make_response, abort
+from datetime import datetime
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+task_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
+
+def validate_model(cls, model_id):
```

Comment:
Nice work refactoring this to be able to validate any model type. You could pull this method into a helper file and import it into routes.py